peps/pep-0772.rst

```diff
@@ -0,0 +1,463 @@
+
+--------------------------
+Interoperability Standards
+--------------------------
+
+The PyPA is responsible for creating and maintaining the interoperability
+standards for Python packaging tools. The decision making for updating these
+standards involves a standing delegation from the Python Steering Council to
+specific individuals based on specific topics, and these individuals have the
+ability to further delegate the decision making on the changes to other
+individuals.
+
+We know that this process is not sustainable and there has been a desire
+expressed by many to have a designated body to make these decisions (including
+the individuals with the standing delegations).
+
+-----------------------
+Python Steering Council
+-----------------------
+
+While there is overlap between the PyPA, Packaging-WG as well as the Python
+core team, the Steering Council is not well-placed to make decisions over
+Python packaging matters directly. Packaging only tangentially intersects with
+Python language evolution, packaging requires specialised expertise and deep
+domain knowledge (much like typing or the C API), and there is a different
+cohort of potential electors for a Packaging Council than the Steering Council.
+
+------------
+Expectations
+------------
+
+An elected Packaging Council that has authority over the interoperability
+standards and Python packaging tooling would have the mandate to coordinate
+efforts around Python packaging. This would provide a better, more consistent
+experience to packagers, package consumers, tool developers, PyPI, and other
+index maintainers. With greater transparency and clearer goals via an elected
+Packaging Council, the PSF will also be able to offer more tactical work,
+longer term strategic vision, and fundraising support to all parts of this
+ecosystem.
+
+Some expectations around the formation of a Packaging Council are that they
+will facilitate:
+
+* Work with PSF staff and the newly formed `User Success Workgroup`_ toward
+  improving the user experience around packaging.
+* A more stable packaging ecosystem that is more responsive to community input.
+* Increased transparency and clearer shared goals for the packaging ecosystem.
+* Increased tactical and fundraising support from the PSF, to increase capacity
+  and funding available to packaging tools.
+
+=============
+Specification
+=============
+
+-----------------
+Packaging Council
+-----------------
+
+The Packaging Council will be composed of 5 individuals.
+
+Mandate
+=======
+
+The council shall work to:
+
+* Improve Python packaging's user experience.
+* Maintain the quality and stability of the Python packaging standards.
+* Formalise and maintain the relationship with the core team as well as the
+  PSF.
+* Establish appropriate decision-making processes.
+* Seek consensus among contributors before acting in a formal capacity.
+* Make contributing as accessible, inclusive, and sustainable as possible.
```

Comment:
I suggest that you order these points in priority of short term and long term efforts. Perhaps: 
```suggestion
* Maintain the quality and stability of the Python packaging standards.
* Formalise and maintain the relationship with the core team as well as the
  PSF.
* Establish appropriate decision-making processes.
* Improve Python packaging's user experience.
* Make contributing as accessible, inclusive, and sustainable as possible.
* Strive to seek consensus among contributors before acting in a formal capacity.
```